src/System.Private.Uri/tests/FunctionalTests/IriEncodingDecodingTests.cs

```diff
@@ -0,0 +1,145 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Text;
+
+using Xunit;
+
+namespace System.PrivateUri.Tests
+{
```

Comment:
Do any of these tests need to be skipped for NETFX (pending onboarding of 4.7.2)?

Reply:
All of them do! Thanks for the reminder.

Reply:
This should now be fixed in the latest commit.